### ISSUE TITLE
Parallel downloads

### ISSUE THREAD

**Issue by MacGritsch** [libcpr/cpr#154]
Hi all,

is there a way to download multiple files in parallel but in one thread (not one thread for each download).

in libcurl there is the "multi" interface: https://curl.haxx.se/libcurl/c/libcurl-multi.html

thank you and best regards

**Comment by COM8** [comment]
Closing in favour of #556
